system-variables.md

```diff
@@ -1031,34 +1081,36 @@ SET tidb_query_log_max_len = 20
 ### tidb_retry_limit
 
 - Scope: SESSION | GLOBAL
-- Default value: 10
+- Default value: `10`
+- Range: `[-1, 9223372036854775807]`
 - This variable is used to set the maximum number of the retries for optimistic transactions. When a transaction encounters retryable errors (such as transaction conflicts, very slow transaction commit, or table schema changes), this transaction is re-executed according to this variable. Note that setting `tidb_retry_limit` to `0` disables the automatic retry. This variable only applies to optimistic transactions, not to pessimistic transactions.
 
 ### tidb_row_format_version
 
 - Scope: GLOBAL
 - Default value: `2`
+- Range: `[1, 2]`
 - Controls the format version of the newly saved data in the table. In TiDB v4.0, the [new storage row format](https://github.com/pingcap/tidb/blob/master/docs/design/2018-07-19-row-format.md) version `2` is used by default to save new data.
 - If you upgrade from a TiDB version earlier than 4.0.0 to 4.0.0, the format version is not changed, and TiDB continues to use the old format of version `1` to write data to the table, which means that **only newly created clusters use the new data format by default**.
 - Note that modifying this variable does not affect the old data that has been saved, but applies the corresponding version format only to the newly written data after modifying this variable.
 
 ### tidb_scatter_region
 
 - Scope: GLOBAL
-- Default value: OFF
+- Default value: `OFF`
 - By default, Regions are split for a new table when it is being created in TiDB. After this variable is enabled, the newly split Regions are scattered immediately during the execution of the `CREATE TABLE` statement. This applies to the scenario where data need to be written in batches right after the tables are created in batches, because the newly split Regions can be scattered in TiKV beforehand and do not have to wait to be scheduled by PD. To ensure the continuous stability of writing data in batches, the `CREATE TABLE` statement returns success only after the Regions are successfully scattered. This makes the statement's execution time multiple times longer than that when you disable this variable.
 
 ### tidb_skip_ascii_check <span class="version-mark">New in v5.0</span>
```

Comment:
Should we link to the similar setting for UTF-8?

Reply:
Thanks! I will do it in a separate PR. For this I'm aiming for parity through an auto-generator.